main.py

```diff
@@ -0,0 +1,176 @@
+import sys
+from csv import reader
+import datetime
+from math import ceil
+import os
+from statistics import mean
```

Comment:
As mentioned before it's a PEP8 violation.
PEP8 asks to group imports into three sections and imports are sorted alphabetically in those.